### PR TITLE
Feat/Bundle product basic implementation

### PR DESCRIPTION
**Pull request by macrozone**
(based on previous pr,sorry :-()

### CODE REVIEW

**Comment by macrozone** [comment]
woops, i accidentially added stuff in controlpanel, sorry

File: packages/api/schema/types/product/bundle-product.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+    type ProductBundleItemConfigurationParameter {
+      key: String!
+      value: String!
+    }
+
+    type ProductBundleItem {
+      product: Product!
+      quantity: Int!
+      configuration: [ProductBundleItemConfigurationParameter!]
```
Comment (pozylon) [inline packages/api/schema/types/product/bundle-product.js]: Just rename ProductItemConfigurationParameter to ProductItemConfigurationParameterInput and use ProductItemConfigurationParameter here.

**Comment by pozylon** [comment]
Oh und please rebase after merge of #12

**Comment by macrozone** [comment]
@pozylon can you do the rebase, you refactored the graphql types and its now hard to resolve the conflicts

**Comment by pozylon** [comment]
@macrozone easy, machi

**Comment by pozylon** [comment]
Schema changes:

OrderItemConfiguration -> ProductConfigurationParameter
ProductBundleItemConfigurationParameter -> ProductConfigurationParameter
ProductBundleItemConfigurationParameterInput -> ProductConfigurationParameterInput

**Pull request merged**